src/invidious/routing.cr

```diff
@@ -131,7 +131,7 @@ module Invidious::Routing
 
     # Channel catch-all, to redirect future routes to the channel's home
     # NOTE: defined last in order to be processed after the other routes
-    get "/channel/:ucid/*", Routes::Channels, :home
+    get "/channel/:ucid/*", Routes::Channels, :redirect_home
```

Comment:
What's the point of using a new handler here?

Reply:
To actually redirect to `/channel/UCxxx`.
The `home` handler displays the video page, but never redirects.
So if you go to `/channel/UCxxx/invalid` the URL won't change and the videos are still displayed, which could confuse users.